.github/workflows/build.yaml

```diff
@@ -139,4 +139,4 @@ jobs:
           sudo atest service install
           sudo atest service restart
       - name: Test
-        run: make test-ui
+        run: make test-ui
```

Comment:
I prefer to keep an empty line in each file.

https://stackoverflow.com/questions/2287967/why-is-it-recommended-to-have-empty-line-in-the-end-of-a-source-file

Below icon indicates there is no new line at the end.
![image](https://github.com/LinuxSuRen/api-testing/assets/1450685/99477a65-87f4-43cb-882e-5a12bfc48087)